staging/src/k8s.io/apiserver/pkg/server/options/authorization.go

```diff
@@ -186,6 +186,7 @@ func (s *DelegatingAuthorizationOptions) getClient() (kubernetes.Interface, erro
 	// set high qps/burst limits since this will effectively limit API server responsiveness
 	clientConfig.QPS = 200
 	clientConfig.Burst = 400
+	clientConfig.Timeout = 10 * time.Second
```

Comment:
Given an overall default timeout on requests of 30s, this amount of time seems reasonable.

Can we add an option (not a CLI flag, but an option in DelegatingAuthorizationOption) to allow setting it?

Reply:
done.